src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms, color 300ms;
```

Comment:
You forgot about styles properties order

![image](https://user-images.githubusercontent.com/110456763/186639708-d86d1b07-e235-4718-9349-3ad08e225ae7.png)